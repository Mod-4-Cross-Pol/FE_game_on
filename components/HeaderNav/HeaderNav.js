import React from 'react';
import { StyleSheet, View, Text, TouchableOpacity } from 'react-native';

export function HeaderNav({ navigation }) {
  return (
    <View style={styles.nav}>
      <View style={styles.navButtonContainer}>
        <TouchableOpacity style={styles.navButton} onPress={() => navigation.navigate('DatePicker')}><Text style={styles.buttonText}>CHANGE DATE</Text></TouchableOpacity>
        <TouchableOpacity style={styles.navButton} onPress={() => navigation.navigate('CreateEventForm')}><Text style={styles.buttonText}>CREATE EVENT</Text></TouchableOpacity>
      </View>
    </View>
  );
}

const styles = StyleSheet.create({
  nav: {
    width: '100%',
    height: 'auto',
    backgroundColor: '#0060b4'
  },
  navButtonContainer: {
    flexDirection: 'row',
    justifyContent: 'space-around',
    alignItems: 'center',
    marginTop: 3
  },
  navText: {
    fontSize: 35,
    textAlign: 'center',
    marginTop: 70
  },
  navButton: {
    backgroundColor: '#FFF',
    height: 50,
    width: '40%',
    borderRadius: 25,
    margin: 10,
    backgroundColor: '#28BFBD',
    borderWidth: 6,
    borderColor: '#dee500',
    justifyContent: 'center',
    alignItems: 'center'
  },
  buttonText: {
    fontSize: 18
  }
});

export default HeaderNav;
