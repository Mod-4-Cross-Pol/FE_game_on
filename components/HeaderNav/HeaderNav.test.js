import React from 'react';
import { HeaderNav } from './HeaderNav';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
Enzyme.configure({ adapter: new Adapter() });

describe('HeaderNav', () => {

  describe('Renders a HeaderNav component', () => {

  let wrapper;

  beforeEach(() => {
    wrapper = Enzyme.shallow(<HeaderNav />);
  })

  test('renders the component', () => {
    expect(wrapper).toMatchSnapshot();
  });

  test('verfy testing file works', () => {
    expect(true).toBeTruthy();
  });

  });

});
