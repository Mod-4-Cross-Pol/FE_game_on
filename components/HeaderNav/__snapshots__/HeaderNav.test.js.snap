// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderNav Renders a HeaderNav component renders the component 1`] = `ShallowWrapper {}`;
